### PR TITLE
Pass entity tags as features to second level CRFs in DIETClassifier

### PR DESCRIPTION
**Pull request by tabergma**
**Proposed changes**:
Pass entity tags instead of logits as features to second level CRFs in DIETClassifier.

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: rasa/nlu/classifiers/diet_classifier.py [hunk 1]
```diff
@@ -1342,6 +1342,14 @@ def _prepare_entity_recognition_layers(self) -> None:
             self._tf_layers[f"crf.{name}"] = layers.CRF(
                 num_tags, self.config[REGULARIZATION_CONSTANT], self.config[SCALE_LOSS]
             )
+            if name == ENTITY_ATTRIBUTE_TYPE:
+                num_entity_tags = num_tags
+            else:
+                self._tf_layers[f"embed.{name}.tags"] = layers.Embed(
```
[the remark this answers is not in the document]
Comment (Ghostvv) [reply rasa/nlu/classifiers/diet_classifier.py:1348]: but this embedding is the one that we'll be concatenated with transformer output, it doesn't have to have num tags dimension, I suggest to use `embed_dim` here

[the remark this answers is not in the document]
Comment (tabergma) [reply rasa/nlu/classifiers/diet_classifier.py:1348]: good idea!
